src/js/node/http.ts

```diff
@@ -446,8 +448,21 @@ class Server extends EventEmitter {
   }
 
   address() {
-    if (!this.#server) return null;
-    return this.#server.address;
+    const address = this.#server?.address;
```

Comment:
was this intentional

Reply:
was not intentional https://github.com/oven-sh/bun/commit/01a7f28a6bfc072434a9641784d97dd18882e761#diff-91e6e16a6a18c42ea259bb91cd801b3707bbbf4f8c8acb48cc3df2152c02d1bd sorry